tests/unit/butterfree/clients/test_cassandra_client.py

```diff
@@ -60,10 +68,14 @@ def test_cassandra_get_schema(self, cassandra_client):
 
         assert sanitize_string(query) == sanitize_string(expected_query)
 
-    def test_cassandra_create_table(self, cassandra_client, cassandra_feature_set):
-        cassandra_client.sql = MagicMock()
+    def test_cassandra_create_table(
+        self,
+        cassandra_client: CassandraClient,
+        cassandra_feature_set: List[Dict[str, Any]],
+    ) -> None:
+        cassandra_client.sql = MagicMock()  # type: ignore
 
-        columns = [
+        columns: List[CassandraColumn] = [
```

Comment:
Here too

Reply:
In this case, we have to explicitly tells what type it's because we're using `TypedDict` ([PEP-589](https://www.python.org/dev/peps/pep-0589/)), `mypy` can't infer if that list is of `CassandraColumn` type.
But we don't need the annotation if we pass the `dict` directly to the function like:
```python
def my_func(column: CassandraColumn) -> None:
    ...

my_func({'column_name': 'test', 'type': 'integer', 'primary_key': False})
```

### Why not use `List[Dict[str, Union[str, bool]]`?

When we define the value type as `Union` means the values will only be strings or booleans in the entire dictionary:
```python
my_dict: List[Dict[str, Union[str, bool]] = [{'key_1': 'value', 'key_2': True}]  # Wrong
my_dict: List[Dict[str, Union[str, bool]] = [{'key_1': 'value', 'key_2': 'value'}]  # Right
my_dict: List[Dict[str, Union[str, bool]] = [{'key_1': False, 'key_2': True}]  # Right
```

`TypedDict` helps us to type properly our dictionary!